src/parser/druid/balance/modules/talents/azeritetraits/PowerOfTheMoon.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import { formatNumber, formatPercentage } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import HIT_TYPES from 'game/HIT_TYPES';
+import Analyzer from 'parser/core/Analyzer';
+import AzeritePowerStatistic from 'interface/statistics/AzeritePowerStatistic';
+import SpellLink from 'common/SpellLink';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import StatTracker from 'parser/shared/modules/StatTracker';
+
+const debug = false;
+
+class PowerOfTheMoon extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    statTracker: StatTracker,
+  };
+  /**
+   * Gives you a buff that increases your solar wrath damage by x for 8 seconds
+   * lets track how much damage you gained from it
+   */
+
+  bonusDamage = 0;
+
+  getAbility = spellId => this.abilityTracker.getAbility(spellId);
+
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.POWER_OF_THE_MOON.id);
+    if (!this.active) {
+      return;
+    }
+    this.bonus = this.selectedCombatant.traitsBySpellId[SPELLS.POWER_OF_THE_MOON.id]
+      .reduce((total, rank) => {
+        const [ damage ] = calculateAzeriteEffects(SPELLS.POWER_OF_THE_MOON.id, rank);
+        debug && this.log(`Rank ${rank}, damage ${damage}`);
+        return total + damage;
+      }, 0);
+  }
+
+  on_byPlayer_damage(event){
+      const spellId = event.ability.guid;
+      const versPerc = this.statTracker.currentVersatilityPercentage;
+      let critMod = 1;
+
+      if(spellId !== SPELLS.MOONFIRE_BEAR.id){
```

Comment:
is `MOONFIRE_BEAR` the right spell for this? seems off...

Reply:
If you look at the druid.js spells there is a comment say that it's that one but I'll double check just for safety... It's very strange how blizzard did all the naming imo